src/Cake.Svn/SvnAliases.Info.cs

```diff
@@ -212,7 +217,7 @@ public static SvnInfoResult GetSvnFileInfo(this ICakeContext context, FilePath f
             var svnInformation = new SvnInfo(context.Environment, SvnClientFactoryMethod);
 
             // A file returns only one SvnInfoResult.
-            return svnInformation.GetInfo(filePath, settings ?? new SvnInfoSettings()).Single();
+            return svnInformation.GetInfo(filePath, settings ?? new SvnInfoSettings()).SingleOrDefault() ?? new SvnInfoResult();
```

Comment:
I would return `null` in this case. Otherwise the caller has to inspect the `SvnInfoResult` if it is a valid result or not